Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM golang:1.18.4-buster
+ENTRYPOINT ["/tagger", "--config", "/tmp/tagger.yml"]
+COPY tagger /tagger
+COPY ./packaging/etc/tagger.yml /tmp/tagger.yml
```

Comment:
> A configuration file is also copied into the Dockerfile. ...
> @tjhop off the bat, I wasn't sure how to handle adding ./packaging/etc/tagger.yml to the build properly, what I have now feels sub optimal

Can we leave the config file out of the image? I'd personally rather suggesting that users bind mount in a config file. My recommendation would be to add a section to the `Usage` header in the README with instructions on how to use the image with a mounted config. The application isn't super complex, something like this would probably be good:
```
docker run \
    -e LINODE_TOKEN="asdfasdfasdf" \
    -v /path/to/tagger.yml:/etc/tagger/tagger.yml \
    ghcr.io/tjhop/linode-tagger```